bigcache.go

```diff
@@ -33,6 +33,79 @@ type cacheShard struct {
 	onRemove    func(wrappedEntry []byte)
 }
 
+// EntryInfo holds informations about entry in the cache
+type EntryInfo struct {
```

Comment:
Reason is simple: entryinfo is lazy - you will only parse what you need instead of everything